From: Simon McVittie <smcv@collabora.com>
Date: Wed, 19 Mar 2025 14:20:24 +0000
Subject: docs: Update URL to libsoup

Similar to GNOME/libsoup!444, Teams/Websites/developer.gnome.org!214

Signed-off-by: Simon McVittie <smcv@collabora.com>
Applied-upstream: 3.9.2, commit:310955fc35c79ec33467fdc8046b549bb5fd5aa9
---
 docs/reference/libtracker-sparql/overview.md.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/docs/reference/libtracker-sparql/overview.md.in b/docs/reference/libtracker-sparql/overview.md.in
index c42b22c..a8b0126 100644
--- a/docs/reference/libtracker-sparql/overview.md.in
+++ b/docs/reference/libtracker-sparql/overview.md.in
@@ -66,7 +66,7 @@ TinySPARQL depends on the following libraries:
   [SQLite website](https://sqlite.org).
 - **Soup**: libsoup is an HTTP and HTTP/2 client/server library for GNOME. It uses
   GObjects and the GLib main loop to integrate well with GNOME applications. More
-  information available on the [Soup API documentation](https://libsoup.org/libsoup-3.0/index.html)
+  information available on the [Soup API documentation](https://libsoup.gnome.org/libsoup-3.0/index.html)
 - **LibXML2**: Libxml2 is the XML C parser and toolkit developed for the GNOME project.
   More information at the [libXML2 website](https://gitlab.gnome.org/GNOME/libxml2/-/wikis/home).
 - **JSON-GLib**: JSON-GLib is a library providing serialization and deserialization
